### DISCUSSION TITLE
Allow execute on mount to be empty object when input is `undefined`

### DISCUSSION THREAD

**Discussion by DavidRowthorn** [next-safe-action/next-safe-action#217] [Ideas]
For an action that doesn't take any input, the following produces a type error (using WebStorm), complaining about `input` not being a required field.

```typescript
const { result } = useAction(myAction, {
    executeOnMount: {},
  })
```

The following fixes it:

```typescript
const { result } = useAction(myAction, {
    executeOnMount: { input: undefined },
  })
```

`{ input: undefined }` should be treated as equivalent to `{}`.

**Comment by TheEdoRan** [comment]
Fixed in [v7.6.2](https://github.com/TheEdoRan/next-safe-action/releases/tag/v7.6.2)
